src/core/lib/iomgr/ev_poll_posix.cc

```diff
@@ -1031,6 +1033,9 @@ static grpc_error* pollset_work(grpc_exec_ctx* exec_ctx, grpc_pollset* pollset,
                       pfds[i].fd, (pfds[i].revents & POLLIN_CHECK) != 0,
                       (pfds[i].revents & POLLOUT_CHECK) != 0, pfds[i].revents);
             }
+            if (pfds[i].revents & POLLHUP) {
```

Comment:
Please add a comment here that this is just a mitigation and add a reference to the PR.